starter-code/.gitignore

```diff
@@ -5,5 +5,8 @@ node_modules
 npm-debug.log
 
 # Environment Variables should NEVER be published
-# temporary removed from .gitignore so the PORT stays accessible after clonning the repo
-# .env
+ .env
```

Comment:
Aaron be careful with the space before the `.env` or `.Ds_Store` or any file that you add in the `.gitignore` otherwise you publish those files